scripts/backend_tests.py

```diff
@@ -240,6 +240,9 @@ def _convert_to_test_target(path):
 def main():
     """Run the tests."""
     parsed_args = _PARSER.parse_args()
+    if parsed_args.test_target is not None:
```

Comment:
Merge both the `if` conditions with [this](https://github.com/oppia/oppia/pull/6014/files#diff-ed3af7853d03776aecf2b00cbb956c1dL251) condition.